psalm.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<psalm
+    totallyTyped="true"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://getpsalm.org/schema/config"
+    xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
```

Comment:
Maybe a good idea to also refer to the local schema, as a backup (If Psalm ships with one, i'm not 100% sure it does)

Reply:
This already refers to a local schema

Reply:
My bad, missed the space in there